src/sheets/updateStatus.js

```diff
@@ -0,0 +1,26 @@
+import {updateItem} from './updateItem'
+
+export async function updateStatus(order, items, status) {
+  const addCompanyItemData = (result, sheetName, rowIndex, item) => {
```

Comment:
1. `item` argument is unused, please remove
2.  changing arguments (in this case `result`) is a side effect and a possible surprise to the caller. Please refactor to return the the object instead of pushing it to result.

Reply:
Done. Quite a lot has changed in `updateItem.js` please look at that file again.